samples/addons/zipkin.yaml

```diff
@@ -0,0 +1,39 @@
+#apiVersion: apps/v1
+#kind: Deployment
+#metadata:
+#  name: zipkin
+#  labels:
+#    app: zipkin
+#spec:
+#  selector:
+#    matchLabels:
+#      app: zipkin
+#  template:
+#    metadata:
+#      labels:
+#        app: zipkin
+#      annotations:
+#        sidecar.istio.io/inject: "false"
+#    spec:
+#      containers:
+#      - name: zipkin
+#        image: openzipkin/zipkin-slim:2.20.0
+#        env:
+#          - name: STORAGE_METHOD
+#            value: "mem"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: zipkin
+  name: zipkin
+spec:
+  ports:
+    - port: 9411
+      targetPort: 9411
+      name: http-query
+  selector:
+    # TODO: We need to figure out if we want to use jaeger/zipkin/both.
```

Comment:
Ignore this part for now, this will be reviewed in P&T WG thursday

Reply:
I probably won't attend the P&T WG mtg.  My 2 cents, Jaeger is a whole lot more user friendly than Zipkin from UX perspective.  I know we removed Zipkin at one point but added back.  If we have folks willing to maintain Zipkin integration with Istio as samples I'm cool with that but we need to ensure there are tests associated with Zipkin (or folks who are willing to manually test it for each release) before we can include it as most users I know don't do manual testing with Zipkin.

Reply:
for what it's worth we have tests for zipkin but not Jaeger right now actually. I'll bring up your opinions tomorrow though - thanks!

Reply:
I have updated based on WG decision: jaeger is the primary deployment, with Zipkins optional addition in `extras/`

Reply:
Thanks, good to hear.